src/NUnitFramework/tests/Constraints/RangeConstraintTests.cs

```diff
@@ -87,5 +87,77 @@ public void ShouldThrowExceptionIfFromIsLessThanTo()
         {
             Assert.Throws<ArgumentException>(() => new RangeConstraint( 42, 5 ));
         }
+
+        #if !NETCF && !SILVERLIGHT  && !PORTABLE
```

Comment:
There was a build issue with Silverlight, CF and portable not having Systems.Collections as the issue was dealing with classes that did not implement IComparable or ICompare. So it was affecting my wrapper class for the unit test.

I know that there are plans to completely remove these tags which those are being spun off into different projects?